lib/ClimaCoreTempestRemap/test/exodus.jl

```diff
@@ -0,0 +1,51 @@
+import ClimaCore
+using ClimaCore: Geometry, Meshes, Domains, Topologies, Spaces
+using NCDatasets
+using TempestRemap_jll
+using Test
+using ClimaCoreTempestRemap
+
+OUTPUT_DIR = mkpath(get(ENV, "CI_OUTPUT_DIR", tempname()))
+
+@testset "mesh coordinates ClimaCore > TempestRemap" begin
```

Comment:
are these tests needed given the updated netcdf tests?

Reply:
yes, they check that our maps align.